docs/kubernetes/user-guides/pre-provisioned.md

```diff
@@ -0,0 +1,90 @@
+# Kubernetes Pre-Provisioned Disks User Guide
+
+This guide gives a simple example on how to use this driver with disks that have
+been pre-provisioned by an administrator.
+
+## Install Driver
+
+See [instructions](driver-install.md)
+
+## Pre-Provision a Disk
+
+If you have not already pre-provisioned a disk on GCP you can do that now.
+
+1. Create example PD
+
+```bash
+gcloud compute disks create test-disk --zone=us-central1-c --size=5Gi
+```
+
+## Create Persistent Volume for Disk
+
+1. Create example Zonal Storage Class
+
+```bash
+kubectl apply -f ./examples/kubernetes/demo-zonal-sc.yaml
+```
+
+2. Create example Persistent Volume
+
+**Note:** The `volumeHandle` and `nodeSelectorTerms` values should be generated
+based on the project, zone, and PD name of the disk created. `storage` value
+should be generated based on the size of the underlying disk
+
+**RePD Note:** For RePD the volume handle is of the form:
+`"projects/${PROJECT}/regions/us-central1/disks/test-disk"` and you must add
+another `nodeSelectorTerms` to match the zones of the RePD.
```

Comment:
There's actually 2 ways to do it. IMO the simpler way is to add one more zone to the "values" array.  Would it be more clear if the example looked something like:
```
values:
- us-central1-c
- <2nd zone if Regional PD>
```

Reply:
ack - did not realize this was the case. I just modeled it after what it looked like after dynamic provisioning.